haystack_experimental/evaluation/harness/rag/evaluation_pipeline.py

```diff
@@ -44,8 +44,12 @@ def default_rag_evaluation_pipeline(
         RAGEvaluationMetric.SEMANTIC_ANSWER_SIMILARITY: partial(
             SASEvaluator, model="sentence-transformers/all-MiniLM-L6-v2"
         ),
-        RAGEvaluationMetric.ANSWER_FAITHFULNESS: partial(FaithfulnessEvaluator, raise_on_failure=False),
-        RAGEvaluationMetric.CONTEXT_RELEVANCE: partial(ContextRelevanceEvaluator, raise_on_failure=False),
+        RAGEvaluationMetric.FAITHFULNESS: partial(
```

Comment:
with ruff line length = 120 this should fit on on-line

Reply:
I've leave that for another PR.